### ISSUE TITLE
Adding Individual Emails

### ISSUE THREAD

**Issue by gingerbeardman** [gingerbeardman/stapler#36]
### Discussed in https://github.com/gingerbeardman/stapler/discussions/35

<div type='discussions-op-text'>

<sup>Originally posted by **FetchEOD** August 26, 2024</sup>
I would love to be able to add individual emails to my Stapler (v1.2.0) document by dragging and dropping, but this doesn't seem to be possible from either Mail or MailMate in MacOS Ventura 13.4.  

At the moment I am either printing to PDF or copy/pasting the text into a text file.

Thanks for such a useful app.

Andy</div>

**Comment by kristofer** [comment]
yes, I have thought on this problem with Apple Mail for years. Even tried hard for a while in 2012 or so to get Apple to allow some kind of "just let me copy the damn email byte stream" but to no avail. Has always seemed like it should be a pasteboard type.

The only way I've ever used has been the print-to-PDF route.  (but then, as you know, there is not way to re-constitute it as an email...)

spent some time in the debugger with Stapler trying to catch the drag event of whatever it is that you can drag from mail, but nothing interesting came of it. 

Is there another way to "decode" drag event other than the two in Stapler right now?

**Comment by gingerbeardman** [comment]
Done in 1.3 MAS
